config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -205,7 +204,6 @@ presubmits:
             - --gcp-zone=us-west1-b
             - --ginkgo-parallel=30
             - --provider=gce
-            - --runtime-config=batch/v2alpha1=true
```

Comment:
I'm guessing this is a deprecation?

Reply:
i looked at `pull-kubernetes-e2e-gce` and made it match. this job should be the same as that job. except for the ubuntu OS images and containerd as runtime

Reply:
SGTM!
/lgtm
/approve
/hold cancel